src/mux-analytics.brs

```diff
@@ -355,16 +355,19 @@ function muxAnalytics() as Object
     m._Flag_isPaused = (videoState = "paused")
     if videoState = "buffering"
       m._checkForSeek("buffering")
-      if m._Flag_atLeastOnePlayEventForContent = true
-        m._addEventToQueue(m._createEvent("rebufferstart"))
-        m._Flag_RebufferingStarted = true
-        if m._viewRebufferCount <> Invalid
-          m._viewRebufferCount++
-          if m._viewWatchTime <> Invalid AND m._viewWatchTime > 0
-            m._viewRebufferFrequency! = m._viewRebufferCount / m._viewWatchTime
+      if m._Flag_isSeeking = false
+        if m._Flag_atLeastOnePlayEventForContent = true
+          m._addEventToQueue(m._createEvent("rebufferstart"))
+          m._Flag_RebufferingStarted = true
+          if m._viewRebufferCount <> Invalid
+            m._viewRebufferCount++
+            if m._viewWatchTime <> Invalid AND m._viewWatchTime > 0
+              m._viewRebufferFrequency! = m._viewRebufferCount / m._viewWatchTime
+            end if
           end if
         end if
       end if
+    end if
```

Comment:
This closes `if videoState = "buffering"` and the next line is `else if videoState = "paused"` which causes a compile-time error.
Removing this `end if` fixes that

Reply:
Thank you for catching that, sorry about that mixup. Pushing an updated version now